src/components/ApplicationGroup/Details/TriggerView/BulkCDTrigger.tsx

```diff
@@ -80,8 +81,9 @@ export default function BulkCDTrigger({
                 _unauthorizedAppList[appDetails.appId] = false
                 _CDMaterialPromiseList.push(
                     getCDMaterialList(appDetails.cdPipelineId, appDetails.stageType, abortControllerRef.current.signal)
-                        .then((r) => {
-                            return { materialList: r, appId: appDetails.appId }
+                        .then((data) => {
+                            const { materials, ...rest } = data
+                            return { materialList: data.materials, appId: appDetails.appId, ...rest }
```

Comment:
we can directly use materials instead of data.materials

Reply:
Done